lighthouse-core/report/templates/report-template.html

```diff
@@ -61,15 +61,19 @@
             <h1 class="menu__header-title">Lighthouse</h1>
             <div class="menu__header-version">Version: {{lighthouseVersion}}</div>
           </div>
-          <ul class="menu__nav">
-            {{#each aggregations}}
-            <li class="menu__nav-item">
-              <a class="menu__link" href="#{{nameToLink this.name}}">
-                {{ this.name }}
-              </a>
-            </li>
-            {{/each}}
-          </ul>
+          {{#if_eq reportContext "perf-x" }}
```

Comment:
instead of `otherReports` array, I used `reportsCatalog`. Because the nav needs to know which report is selected currently. Adding a boolean flag to each of the element in the report is another approach, but I feel using `reportsCatalog` is safer, because we won't accidentally mark multiple reports as selected.